core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -150,4 +151,13 @@ export class DotEditPageNavComponent implements OnChanges {
             !template ? 'editpage.toolbar.nav.layout' : 'editpage.toolbar.nav.layout'
         );
     }
+
+    private disableLayoutOnExperimentVariant(): boolean {
```

Comment:
```suggestion
    private isDisableLayoutOnExperimentVariant(): boolean {
```

???

Reply:
in this case I like more `disableLayoutOnExperimentVariant` to keep it as a question based on the conditions inside the function.